analyzers/tests/SonarAnalyzer.UnitTest/TestCases/WebConfig/CookieShouldBeHttpOnly/HttpOnlyCookiesConfig/CookieShouldBeHttpOnly.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
```

Comment:
Also, let's move this file to `TestCases/Hotsposts` to keep all of them together.